packages/core/strapi/lib/core/loaders/plugins/index.js

```diff
@@ -85,7 +85,15 @@ const loadPlugins = async (strapi) => {
   for (const pluginName of Object.keys(enabledPlugins)) {
     const enabledPlugin = enabledPlugins[pluginName];
 
-    const serverEntrypointPath = join(enabledPlugin.pathToPlugin, 'strapi-server.js');
+    let serverEntrypointPath;
+
+    try {
+      serverEntrypointPath = join(enabledPlugin.pathToPlugin, 'strapi-server.js');
+    } catch (e) {
+      throw new Error(
+        `Error loading plugin ${pluginName}, plugin is not installed, please install the plugin or remove it's configuration.`
```

Comment:
```suggestion
        `Error loading the plugin ${pluginName} because ${pluginName} is not installed. Please either install the plugin or remove it's configuration from DERRICK_WHERES_THE_CONFIG.`
```

I don't know off the top of my head where the config is 😅

Reply:
It depends, either ./config/plugins.js or in the environment folder structure (which has unlimited possibilities since  it depends on the NODE_ENV variable. It's exactly why I didn't mention it.

Reply:
shame, would have been more helpful imo.